.github/workflows/slsa-builder-go.yml

```diff
@@ -179,63 +225,67 @@ jobs:
           path: ${{ steps.build-gen.outputs.go-binary-name }}
           retention-days: 5
 
+      #TODO(upload): verify how the action handles malicious names.
+      - name: Upload the provenance
+        uses: actions/upload-artifact@82c141cc518b40d92cc801eee768e7aafc9c2fa2 # v2.3.1
+        with:
+          name: ${{ steps.build-gen.outputs.provenance-unsigned }}
```

Comment:
add `"` around the variables. Mhh, we can do that for all variables in a follow-up PR if you prefer. I've created https://github.com/asraa/slsa-on-github/issues/35 to track it